### PR TITLE
Project Linting

### PR DESCRIPTION
**Pull request by Soitora**
Expands upon the linter tool which reported on new errors that should be fixed. I fixed them all and hid errors for ones I don't want to tamper as they're Vuepress native code warnings.

I did one commit per action/fix so I could easily see what was done where and when, fortunately Squash exists.

### CODE REVIEW

File: src/help/contribution.md [hunk 1]
```diff
@@ -45,8 +45,8 @@ Visit our translation project [here](https://hosted.weblate.org/projects/tachiyo
 </a>
 
 ## Donation
-If you can't contribute code or translations but you still wish to help, then you can choose to contribute directly to the projects founder, [Inorichi](https://github.com/inorichi/) by using the button below, or going [directly to Ko-Fi](https://ko-fi.com/inorichi).
+If you can"t contribute code or translations but you still wish to help, then you can choose to contribute directly to the projects founder, [Inorichi](https://github.com/inorichi/) by using the button below, or going [directly to Ko-Fi](https://ko-fi.com/inorichi).
```
Comment (arkon) [inline src/help/contribution.md:48]: Mistake.

File: src/help/guides/source-migration.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Migration is the process of moving manga from one source to another while keepin
 1. Select the **Source** you're migrating from.
 1. Find and select the title that you want to migrate.
 
-   **Tachiyomi** will do a global search of all the sources you have installed and enabled. If you don't find the manga you want, try searching for other titles it might go under.
+	**Tachiyomi** will do a global search of all the sources you have installed and enabled. If you don't find the manga you want, try searching for other titles it might go under.
```
Comment (arkon) [inline src/help/guides/source-migration.md:16]: Mistake? I feel like you shouldn't be applying indentation rules on markdown files.

File: src/help/contribution.md (hunk 1 above)
Comment (Soitora) [reply src/help/contribution.md:48]: Can't check until tomorrow, Fasthub does not show any difference.

File: src/help/guides/source-migration.md (hunk 2 above)
Comment (Soitora) [reply src/help/guides/source-migration.md:16]: Intentional, but idm the indentation for md content that much

**Pull request merged**